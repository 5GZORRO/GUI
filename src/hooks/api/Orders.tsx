import { useMutation, useQuery } from 'react-query'
import { api } from 'api'

export const useMyOrders = (params?: any) =>
  useQuery(['myOrders', params], () => api.orders.getMyOrders(params), { keepPreviousData: true })

export const useCreateOrder = () => useMutation(['createOrder'], (body: any) => api.orders.createOrder(body))

export const getOrderedItems = (params?: any) =>
  useQuery(['orderedItems', params], () => api.orders.getOrderedItems(params), { keepPreviousData: true })

export const getAllProductOrders = (params?: any) =>
  useQuery(['getAllProductOrders', params], () => api.orders.getAllProductOrders(params), { keepPreviousData: true })
