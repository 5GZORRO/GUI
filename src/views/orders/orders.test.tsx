// @ts-nocheck
import React from 'react'
import TestRenderer from 'react-test-renderer'
import Orders from './index'

describe('Orders', function () {
  it('Snapshot ', () => {
    const tree = TestRenderer.create(
        <Orders />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
