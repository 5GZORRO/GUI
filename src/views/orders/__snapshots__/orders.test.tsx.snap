// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Orders Snapshot  1`] = `
<h1>
  Orders
</h1>
`;
